packages/lesswrong/lib/collections/localgroups/schema.ts

```diff
@@ -167,6 +167,18 @@ const schema: SchemaType<DbLocalgroup> = {
     regEx: SimpleSchema.RegEx.Url,
     tooltip: 'https://www.meetup.com/...'
   },
+  
+  slackLink: {
```

Comment:
Also added the "Slack Workspace" field for groups, since many online groups are primarily hosted there